src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEventSummary.kt

```diff
@@ -356,14 +390,59 @@ object HoppityEventSummary {
         add(cardRenderable)
     }
 
+    private fun buildTitle(statYear: Int) = Renderable.verticalContainer(
+        buildList {
+            addString(
+                "§dHoppity's Hunt #${getHoppityEventNumber(statYear)} Stats",
+                horizontalAlign = RenderUtils.HorizontalAlignment.CENTER,
+            )
+            val eventEnd = statYear.getEventEndMark()
+            val yearNow = getCurrentSBYear()
+            val isHoppity = HoppityAPI.isHoppityEvent()
+
+            val isCurrentEvent = isHoppity && statYear == yearNow
+            val isPastEvent = statYear < yearNow || (statYear == yearNow && !isHoppity)
+            val isNextEvent = statYear > yearNow
+
+            when {
```

Comment:
please find a more compact way to fix code duplication![image](https://github.com/user-attachments/assets/a622d18f-b474-43cb-bb50-f9a6e5d82db0)